### PR TITLE
Show trophy inline

### PR DESCRIPTION
**Pull request by richychn**
Added Trophy component to show trophy if someone has already minted. 

Styling/placement of image is not great, so welcome feedback there. 

image

### CODE REVIEW

File: pages/api/polygon.ts [hunk 1]
```diff
@@ -171,11 +173,18 @@ export default async function handler(req, res) {
       process.env.POLYGON_API_KEY
   );
   const trophyJson = await trophyResponse.json();
-
+  console.log(trophyJson);
```
Comment (epan) [inline pages/api/polygon.ts:176]: Probably meant to delete this?

File: components/Trophy.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                <p>
+                    Hurry! There are only 10,000 trophies. The first 333 winners get a Gold trophy, the next 3,333 get a Silver trophy, and the rest get a Bronze trophy.
+                </p>
+            </>
+        )
+    } else {
+        return (
+            <>
+                <p>Here's your trophy:</p>
+                <img src={'/fweb_yearone_' + props.trophyColor + '.png'} style={{width: '20vh'}} />
```
Comment (slavingia) [inline components/Trophy.tsx:23]: I think wrapping this in a <p> will help w/ spacing.

Comment (richychn) [reply components/Trophy.tsx:23]: Not quite sure what you mean here. I tried wrapping in div and span, but changes nothing.

File: components/Trophy.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const Trophy = (props) => {
+    if (props.trophyColor == '') {
+        return (
+            <>
+                <p>
+                    For your efforts, you&apos;ve received 1,000 FWEB3 tokens and can now mint a{" "}
+                    <a href="https://polygonscan.com/address/0x2a0493dee4f4b5e4b595326f0e73645f6f493923#writeContract">
```
[the remark this answers is not in the document]
Comment (richychn) [reply components/Trophy.tsx]: Also, not 100% sure what you mean here. I edited to have the entire sentence in one line, but that's probably not what you meant.

File: pages/api/polygon.ts (hunk 1 above)
Comment (richychn) [reply pages/api/polygon.ts]: thanks for catching this!

File: components/Trophy.tsx (hunk 2 above)
Comment (slavingia) [reply components/Trophy.tsx:21]: I wrote a paragraph tag that got hidden lol.

**Review by slavingia — COMMENTED** [review]
Can you merge main into this? Can merge after that

File: components/Trophy.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                <p>
+                    Hurry! There are only 10,000 trophies. The first 333 winners get a Gold trophy, the next 3,333 get a Silver trophy, and the rest get a Bronze trophy.
+                </p>
+            </>
+        )
+    } else {
+        return (
+            <>
+                <p>Here's your trophy:</p>
+                <img src={'/fweb_yearone_' + props.trophyColor + '.png'} style={{width: '20vh'}} />
```
Comment (slavingia) [inline components/Trophy.tsx:21]:
```suggestion
                <p>
                  <img src={'/fweb_yearone_' + props.trophyColor + '.png'} style={{width: '100%'}} />
                </p>
```

File: components/Trophy.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+                    Hurry! There are only 10,000 trophies. The first 333 winners get a Gold trophy, the next 3,333 get a Silver trophy, and the rest get a Bronze trophy.
+                </p>
+            </>
+        )
+    } else {
+        return (
+            <>
+                <p>Here's your trophy:</p>
+                <p>
+                  <img src={'/fweb_yearone_' + props.trophyColor + '.png'} style={{width: '100%'}} />
```
Comment (slavingia) [inline components/Trophy.tsx:22]: Let's also link this to the OpenSea listing

File: components/Trophy.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+                </p>
+                <p>
+                    Hurry! There are only 10,000 trophies. The first 333 winners get a Gold trophy, the next 3,333 get a Silver trophy, and the rest get a Bronze trophy.
+                </p>
+            </>
+        )
+    } else {
+        return (
+            <>
+                <p>Here's your trophy:</p>
```
Comment (slavingia) [inline components/Trophy.tsx:20]:
```suggestion
                <p>Here&apos;s your trophy:</p>
```

File: components/GameFinish.tsx [hunk 7]
```diff
@@ -299,7 +300,29 @@ const GameFinish = () => {
         </button>
       </>
     );
-    // User is verified and is not a winner and is on their own page
```
Comment (slavingia) [inline components/GameFinish.tsx:302]: think i messed this up in a merge conflict : /

File: pages/api/polygon.ts [hunk 8]
```diff
@@ -175,6 +177,13 @@ export default async function handler(req, res) {
     let transaction = trophyJson.result[i];
     if (transaction.from === "0x0000000000000000000000000000000000000000") {
       hasWonGame = true;
+      if (transaction.tokenID <= 333) {
+        trophyColor = 'gold';
+      } else if (transaction.tokenID <= 3333) {
+        trophyColor = 'silver';
+      } else {
+        trophyColor = 'copper'
```
Comment (slavingia) [inline pages/api/polygon.ts:185]:
```suggestion
        trophyColor = 'copper';
```

File: pages/api/polygon.ts [hunk 9]
```diff
@@ -189,5 +198,6 @@ export default async function handler(req, res) {
     hasVotedInPoll: hasVotedInPoll,
     hasDeployedContract: hasDeployedContract,
     hasWonGame: hasWonGame,
+    trophyColor: trophyColor
```
Comment (slavingia) [inline pages/api/polygon.ts:201]:
```suggestion
    trophyColor: trophyColor,
```

**Pull request merged**